src/test/java/org/jabref/logic/importer/fetcher/GrobidCitationFetcherTest.java

```diff
@@ -26,37 +26,37 @@
     static GrobidCitationFetcher grobidCitationFetcher = new GrobidCitationFetcher(importFormatPreferences);
 
     static String example1 = "Derwing, T. M., Rossiter, M. J., & Munro, M. J. (2002). Teaching native speakers to listen to foreign-accented speech. Journal of Multilingual and Multicultural Development, 23(4), 245-259.";
-    static BibEntry example1AsBibEntry = new BibEntry(StandardEntryType.Article).withCiteKey("-1")
-                .withField(StandardField.AUTHOR, "Derwing, T and Rossiter, M and Munro, M")
-                .withField(StandardField.TITLE, "Teaching native speakers to listen to foreign-accented speech")
-                .withField(StandardField.JOURNAL, "Journal of Multilingual and Multicultural Development")
-                .withField(StandardField.YEAR, "2002")
-                .withField(StandardField.PAGES, "245--259")
-                .withField(StandardField.VOLUME, "23")
-                .withField(StandardField.NUMBER, "4");
+    static BibEntry example1AsBibEntry = new BibEntry(StandardEntryType.Article).withCitationKey("-1")
+                                                                                .withField(StandardField.AUTHOR, "Derwing, T and Rossiter, M and Munro, M")
```

Comment:
Why is this reformatted?

Reply:
Because Intellj thinks this is the way it should look like, and I agree